config/rbac/sg_role.yaml

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: smart-gateway-role
+rules:
+  - apiGroups:
```

Comment:
This first rule (allow create on SARs) looks redundant given the second rule (allow create on SARs and tokenreviews)

Reply:
This was actually wrong. The subjectaccessreview is part of the authorization API not the authentication API. I actually had this wrong and "fixed" it, but forgot to remove the superfluous subjectaccessreview from the invalid API group. Resolved!